guides/_subsections/add-a-trusted-certificate.md

```diff
@@ -0,0 +1,30 @@
+## [Add a Trusted Certificate](#-add-a-trusted-certificate)
+If you have Java Management Extensions (JMX) over SSL enabled on your containerized JVMs, you must configure Cryostat to trust the SSL certificate presented by the containerized JVM when Cryostat attempts to open a JMX connection. If this configuration is not completed then Cryostat will be unable to open the connection and unable to perform JFR management tasks.
+
+Here's how to add a trusted SSL certificate with the Cryostat web UI.
+
+<ol>
+    <li>
+        {% include howto_step.html
+          summary="Navigate to the Security tab"
+          image-name="store-jmx-credentials-1.png"
+          text="
+            First, navigate to the <i>Security</i> tab. The Import SSL Certificates card contains an <i>Upload</i> button. Click <i>Upload</i> to summon a file upload dialog and choose the certificate you wish to upload. You may repeat this process multiple times to add multiple trusted certificates.
+          "
+        %}
+    </li>
+    <li>
+        Restart Cryostat to apply the changes. If Cryostat is not restarted then the newly-added certificates are not reloaded and trusted by the Cryostat JMX client and connections will continue to fail. Depending on your deployment platform and configuration, restarting Cryostat might entail any of the following:
+        <ul>
```

Comment:
I am thinking we should add some context around each command? Like, `On OpenShift`, `With Podman` maybe?

Reply:
Also, `my-cryostat-name` could be more specific. For example, with podman, maybe it should be containerID/name? And on OCP, it could be cryostat CR name/deploymentConfig name?

Reply:
Good ideas.

Reply:
Consider:

> 3. Restart Cryostat to apply the changes. If you do not restart your Cryostat instance, the added certificates are not reloaded. This causes connections to fail because the Cryostat JMX client cannot trust the certificates. Depending on your deployment platform and configuration, restarting Cryostat might lead to the following changes:

Reply:
I slightly reworded the last part here but most of this is good.